exporter/sapmexporter/factory.go

```diff
@@ -53,6 +55,18 @@ func createDefaultConfig() configmodels.Exporter {
 		TimeoutSettings: exporterhelper.CreateDefaultTimeoutSettings(),
 		RetrySettings:   exporterhelper.CreateDefaultRetrySettings(),
 		QueueSettings:   qs,
+		Correlation: CorrelationConfig{
+			Enabled:             true,
```

Comment:
If this does always default to `true` then that means anybody who upgrades without providing the API base url config will get a bunch of errors until they do, right?  Something at least needs to validate that the API url is provided and cause only a one time error.

Reply:
Right. I've added validation.